### ISSUE TITLE
difference between channel-wise loss and pixel loss. （3 question）

### ISSUE THREAD

**Issue by Ttmx11** [irfanICMLL/TorchDistiller#9]
1 Is the difference between channel-wise and pixel-wise loss related to the dimensionality of their KLDivLoss calculation? One operates on the width and height dimensions, while the other operates on the channel dimension.

2 I noticed that both the feature map and the logits map are used for channel loss calculation in the paper, but I only found the calculation at one specific location before the network output. Could you please explain the specific calculation process?

3 I have implemented your method, along with two types of loss and the pixel loss from the previous paper "Structured Knowledge Distillation for Semantic Segmentation". However, during network training, all three losses are increasing, despite adjusting the batch size. Could you please suggest any other possible reasons for this behavior?

Thank you very much!!!
